rest/models/UserIdentity.go

```diff
@@ -0,0 +1,14 @@
+package models 
+import (
+  "time"
+)
+
+type UserIdentity struct {
+  BaseModel
+  AccountId string
+  FirstLoging bool
+  DayOne bool
+  LastLogin time.Time
+  LastVisitedPages []LastVisitedPage `gorm:"many2many:user_last_visited_pages"`
```

Comment:
These will not be many to many relations. Each last visited page entry will be only for one specific user. Same goes for favorite pages.

It should be one to many relation.